packages/SwingSet/src/vats/timer/types.js

```diff
@@ -1,5 +1,53 @@
 // @ts-check
 
+// TODO ts seems confused about whether I need to import ERef or not.
+// If I do, then ts in the browser complains. If I don't, then
+// `yarn lint` complains.
+// @template T @typedef {import('@endo/far').ERef<T>} ERef */
+// I get similar symptoms for Notifier
+
+/**
+ * @typedef {object} TimerBrand
+ * @property {(timer: TimerService) => ERef<boolean>} isMyTimer
+ */
+
+/**
+ * @typedef {bigint} TimestampValue
```

Comment:
Not really. We still need a name for these types, i.e., the bigint that `relValue` and `absValue` returns. The thing that we need to "deprecate" is the scalar arm of `Timestamp` and `RelativeTime`, leaving only the contents of what's currently called `TimestampRecord` and `RelativeTimeRecord`. But even names these are sensible to use during the transition, to record that a particular bit of code is only the other side. But once the transition is done those uses will get renamed to the simpler names.

So I don't know how to deprecate what we'd like to deprecate. @turadg , care to advise?

In any case, for this PR `TimeMath` now has an adequate comment explaining the transition plan and representation issues.

Reply:
> The thing that we need to "deprecate" is the scalar arm of Timestamp and RelativeTime,

Agree those should be deprecated with `@deprecate` comment.

> We still need a name for these types, i.e., the bigint that `relValue` and `absValue` returns

I'm not sure of this. Since TypeScript is structurally typed, these `TimestampValue` and `RelativeTimeValue` are only useful for documentation. Branded times is a better solution and reduces the value of these types such that they're not worth the confusion they introduce. Confusion being that they are actually the same type and passing them around like this may give a false sense of type safety.

If you want to brand the bigint like this then I think they're worth keeping around,
https://github.com/Agoric/agoric-sdk/blob/6b0deac341424da9b1e8badfbf307c37f16b4618/packages/zoe/src/zoeService/utils.d.ts#L18-L23
But I expect that when a time value brand matters it will pass around the branded object and only extract the bigint when the brand is unambiguous.

Reply:
Verbally discussed with @turadg and @Chris-Hibbert . Conclusion was not to address the deprecation issue in this PR. Experimenting, we did not seem to be able to deprecate a type. Resolving this conversation and moving forward with this PR.